browser/src/control/jsdialog/Widget.LanguageSelector.js

```diff
@@ -1,4 +1,9 @@
 /* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
 /*
  * JSDialog.LanguageSelector - widgets for selecting spell/grammar chacking language
```

Comment:
I suspect we could make this more readable by having the brief description of the file just under the emacs indenting thing - and merging these two comments into one so:
   ```
/*
    * JSDialog.. ... -
    *
    * Copyright ...
    *
    *  SPDX
    */
```
Hopefully that saves us two vertical lines while looking nice.

Also s/chacking/checking/ on this particular line =)

Otherwise great - thanks!